pkg/ibmvpc/analysis_output_test.go

```diff
@@ -139,7 +139,9 @@ func getTestFileSuffix(format vpcmodel.OutFormat) (suffix string, err error) {
 // initTest: based on the test name, set the input config file name, and the output
 // files names (actual and expected), per use case
 func (tt *vpcGeneralTest) initTest() {
-	tt.inputConfig = inputFilePrefix + tt.name + ".json"
+	if tt.inputConfig == "" {
```

Comment:
what is the scenario in which `tt.inputConfig` is empty?

Reply:
in analysis_output_tests we do not pass the input_config.. we get it from name(which is uniqe for each test) but in explain tests it is not

Reply:
can this be changed to be unique for both `analysis_output_tests` and `explainability` tests? (not necessarily in this PR, can do this as a followup).